check-before-deployment.md

```diff
@@ -192,17 +192,19 @@ To check whether the NTP service is installed and whether it synchronizes with t
     - If it returns `Unit ntpd.service could not be found.`, then try the following command to see whether your system is configured to use `chronyd` instead of `ntpd` to perform clock synchronization with NTP:
 
         {{< copyable "shell-regular" >}}
-    
+
         ```bash
-        sudo systemctl status cronyd.service
+        sudo systemctl status chronyd.service
         ```
-    
+
         ```
         chronyd.service - NTP client/server
         Loaded: loaded (/usr/lib/systemd/system/chronyd.service; enabled; vendor preset: enabled)
         Active: active (running) since Mon 2021-04-05 09:55:29 EDT; 3 days ago
         ```
-    
+
+        If your system configures no `chronyd` or `ntpd`, it means your system installs neither of them. You should first install `chronyd` or `ntpd` and ensure that it can be automatically started. By default, `ntpd` is used.
```

Comment:
```suggestion
        If the result shows that neither `chronyd` nor `ntpd` is configured, it means that neither of them is installed in your system. You should first install `chronyd` or `ntpd` and ensure that it can be automatically started. By default, `ntpd` is used.
```